README.md

```diff
@@ -51,6 +51,10 @@ this.logger.log({ id: 'your message here' }, 'context-name');
 
 ```
 
+### Authentication
+The Master Data Management API requires an API Key in order to access its resources.
```

Comment:
Can we change `Master Data Management` to `Trade Finance Services` please?

Reply:
good spot - I copy pasted that!

Reply:
this is updated now